docs/Guides/Ecosystem.md

```diff
@@ -152,6 +152,15 @@ section.
 
 #### [Community](#community)
 
+> 🛈 Note:
```

Comment:
I have a feeling this is not rendered correctly.

Reply:
See the actual rendering 
<img width="1034" alt="fastify - Vivaldi 2025-05-20 06-24-26" src="https://github.com/user-attachments/assets/ebd55094-e855-4fb7-8e9e-8730889ee025" />

Reply:
> See the actual rendering

Bloody hell, looks fine in Firefox and Edge:
![image](https://github.com/user-attachments/assets/00d97c9f-449b-45ca-8390-dddfa5482b29)

But when looking at it on Safari on iPhone it looks like your example.

Reply:
Use ℹ️?